src/Microsoft.SqlTools.ServiceLayer/Migration/Contracts/MigrationAssessmentsRequest.cs

```diff
@@ -16,20 +17,21 @@ public class MigrationAssessmentsParams
     public class MigrationAssessmentResult
     {
         /// <summary>
-        /// Gets the collection of assessment results.
+        /// Errors that happen while running the assessment
         /// </summary>
-        public List<MigrationAssessmentInfo> Items { get; } = new List<MigrationAssessmentInfo>();
-
+        public ErrorModel[] Errors { get; set; }
         /// <summary>
-        /// Gets or sets a value indicating
-        /// if assessment operation was successful.
+        /// Result of the assessment
         /// </summary>
-        public bool Success { get; set; }
-
+        public ServerProperties Result { get; set; }
```

Comment:
It seems a little odd to have the "result" of an assessment be a set of server properties. It seems like it'd be better to have the result be a boolean and then have a separate property that contains the server properties